docs/meetings/client/2022_10_14.md

```diff
@@ -0,0 +1,83 @@
+- **Date:** October 14, 2022
+
+## Attendance
+
+- Client:
+  - Nazim Ragimov (CEO)
+  - Erik (developer)
+
+- Team A:
+  - Edouard Eltherington
+  - Veronica Jack
+  - Logan Parker
+
+***Attendence Comments***
+
+- Matt Kuelker is writing a midterm, so he was unable to attend
+- Students from other teams present as well
+  - Team B only had 1 group member present
+
+## Agenda Items
+
+- Requirements Confirmation
+- Access Discussion
+- Milestone Discussion
+
+### TEAM A
+
+- We explained that our approach will work for user-selected avatars, but it unfortunately won't work for user-uploaded
+ images. Nazim and Erik said that this is acceptable.
+
+- We went over our DFDs (level 0 and level 1) to ensure that the requirements we understood are correct, and both Nazim
+and Erik confirmed that we are on the right track.
+
+- We asked about access to making requests from the Kukarella system to receive the audio file, language, and user text.
+There currently is no Kukarella API for us to call, so we will need some other way to access the audio, language, and text.
+
+- We asked if there are currently any 2D assets that we can have access to. The image assets that they currently have are
+animation files from their other app, so they're not still images and they do NOT have mouth images at this time
+
+- We will be requesting the following by (date TBD at next team meeting)
+  - Instructions for how to receive the audio file, language, and user text (Kukarella API)
+  - Instructions for how to access Kukarella's image asset database
+  - Please make this Kukarella account premium: `team.a.kukarella.capstone@gmail.com`
+
+- Team A deliverable ideas (Final results TBD at next team meeting)
+  - create phoneme file from audio
+  - return an MP4
+  - ensure it is setup as an API (can make calls to our program)
+  - ensure the MP4 has mouth movements changing
+  - make the animation more realistic
+
+### TEAM B
+
+- Team B had the same questions as Team A
+
+### TEAM C
+
+- They asked if the client has or will have a database with the image assets. Erik said that they will be storing the
+images on their end, so we will need to call on their DB for the images once they've been created.
+
+- They asked if output persists on Kukarella's end. Erik said that no cleanup system in place; stored permanently on
+their end.
+
+- They proposed alternate milestones for their group:
+  1. Basic phoneme detection + usage of Kukarella TTS
+  2. Generate a speaking portrait with audio
+  3. API endpoint integration with Kukarella allowing use of preset avatars
+  4. User-friendly method of uploading custom avatars
+
+## To-do List
+
+1. Send Nazim a complied list (all groups together) of what we all need access to and the deadline
+
+2. Go into the Google Sheets file that Nazim shared and request edit access (Veronica received edit access immediately after the meeting)
+
+3. Once you have edit access, adjust the milestone calendar to reflect each team's milestones.
```

Comment:
We have edit access!

Reply:
We do! I kept the third to-do because we need to do that in our team meeting!